workers/unity/Packages/com.improbable.gdk.core/Systems/ComponentSendSystem.cs

```diff
@@ -72,22 +72,17 @@ internal void AddComponentReplicator(IComponentReplicationHandler componentRepli
         private void PopulateDefaultComponentReplicators()
         {
             // Find all component specific replicators and create an instance.
-            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            var types = ReflectionUtility.GetNonAbstractTypes(typeof(IComponentReplicationHandler));
+            foreach (var type in types)
             {
-                foreach (var type in assembly.GetTypes())
+                if (type.GetCustomAttribute(typeof(DisableAutoRegisterAttribute)) != null)
```

Comment:
Why is this one not using the generic version?

Reply:
The generic one just calls this one.

Reply:
Can make it generic to match the other one though.